Carthage-xcfilelist/app-output-files.xcfilelist

```diff
@@ -2,16 +2,13 @@
 
 $(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/Alamofire.framework
 $(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/AlamofireImage.framework
-$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/AppCenter.framework
```

Comment:
Seems we removed this from the input files list but forgot to remove it from the output files list when we added the AppCenter SDK. This shouldn't be required because we don't use AppCenter for our app store builds.

Reply:
Thanks for cleaning this up 👍